### DISCUSSION TITLE
Adding references to an existing project?

### DISCUSSION THREAD

**Discussion by KimBoesen** [asreview/asreview#1414] [Q&A]
Dear AS Review team,

We are working on a continuously updated review and we would like to implement ASReview in our workflow. Since our database will be continuously updated, we are interested in learning how to  add references to an existing project (without having to start over and lose the model training).

It doesn’t seem possible to add references directly to an existing project in AS Review lab. I see one potential solution: to export the (partially) labelled dataset after the first initial screening round (as Excel file), then insert the new references (from the updated search) in the Excel file, and import the file to a new project in AS Review. Would this work? Importantly, if I import this updated/modified file, will AS Review lab recognise the existing labelling (relevant/irrelevant) and will the newly added references be ordered accordingly to the already trained model? If so, this would solve our problem. 

If this method is not possible, are there other ways we can add references to an existing project, while maintaining the trained model?

We look forward to hearing from you and any help will be useful!

**Comment by rohitgarud** [comment]
Hi @KimBoesen I think [Asreview-Datatools](https://github.com/asreview/asreview-datatools) extension and the compose functionality will help you in what you are trying to achieve. If you keep the labels from the previous dataset and add a new dataset using compose, when you import the new dataset, the already labeled records will be taken as prior knowledge and the model will be trained to the previous state. So when you start reviewing, ASReview will show you only the unlabelled records based on the trained model.

**Reply by Rensvandeschoot** [reply]
There is a nice tutorial  how to do this: https://github.com/asreview/asreview-datatools/blob/master/Tutorials.md

**Reply by KimBoesen** [reply]
Thanks so much for your replies, it's much appreciated! We will try the compose method and I will let know how it worked out. 
Thanks again!
